packages/twenty-front/src/modules/context-store/hooks/useContextStoreObjectMetadataItemOrThrow.ts

```diff
@@ -0,0 +1,17 @@
+import { contextStoreCurrentObjectMetadataItemComponentState } from '@/context-store/states/contextStoreCurrentObjectMetadataItemComponentState';
+import { useRecoilComponentValueV2 } from '@/ui/utilities/state/component-state/hooks/useRecoilComponentValueV2';
+
+export const useContextStoreObjectMetadataItemOrThrow = (
```

Comment:
I really think that should be a convention to add "OrThrow" as a suffix because throwing in React is not conventionnal, it should be reminded in the naming.